packages/token-kiosk/src/files/models/lyrxCreator.ts

```diff
@@ -45,11 +45,13 @@ function createLayerDefinition(record: CatalogRecord, token: string): object {
 }
 
 export function createLyrxFile(layers: CatalogRecord[], token: string): object {
+  // ArcPro needs the layers in reverse order unlike QGIS, if not reversed it will display top layer at the botoom
+  const reveresedLayers = layers.reverse();
```

Comment:
Since reverse function [changes the source object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse#return_value), It is better to create a copy of the object.

In addition, fix typo

```suggestion
  const reversedLayers = [...layers].reverse();
```

Reply:
maybe cover this in a test?

Reply:
Fixed, @CptSchnitz. I was confused how the tests are passed if there is a precise snapshot where the layer order is now wrong. I am not familiar enough with JEST snapshots, and need help

Reply:
thanks to ofer I added the needed snapshot for test.